packages/it-multipart/test.js

```diff
@@ -50,6 +60,7 @@ test.before.cb((t) => {
     res.writeHead(404)
     res.end()
   }).listen(() => {
+    // @ts-ignore - could be null
```

Comment:
What could be null?  After `.listen` `server.address()` returns an object with a port.

Reply:
I'm afraid type systems aren't generally able to handle type changes in effect to certain API call. As per nodejs docs:

> `server.address()`
> _Added in: v0.1.90_
>
> - Returns: `<Object> | <string> | <null>`
> .....
> `server.address()` returns `null` before the 'listening' event has been emitted or after calling `server.close()`.
>

Therefor return type of `server.address()` for TS is `null | string | ...`.

I'll update comment to say that TS can't infer this isn't null because it's between `listen()` and `close()` calls.